src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/OrganisationMfaStatusController.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.professionalapi.controller.external;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.professionalapi.controller.SuperController;
+import uk.gov.hmcts.reform.professionalapi.controller.response.MfaStatusResponse;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RequestMapping(
+        path = "/refdata/external/v1/organisations/mfa"
+)
+@RestController
+public class OrganisationMfaStatusController extends SuperController {
```

Comment:
Do we require a new Controller class for this API? Or can we not place this is `OrganisationExternalController`?

Reply:
Yes, @shreedhar-hmcts suggested we create a separate controller for this